spring-cloud-task-docs/src/main/asciidoc/batch.adoc

```diff
@@ -64,11 +64,16 @@ look like the following:
 @Bean
 public PartitionHandler partitionHandler(TaskLauncher taskLauncher,
 		JobExplorer jobExplorer) throws Exception {
-	MavenResource resource =
+
+	MavenProperties mavenProperties = new MavenProperties();
```

Comment:
would it be too much for this example to show the use of `DelegatingResourceLoader` instead of this way of handling the `MavenProperties`?